modules/ROOT/pages/object-store/object-store-connector.adoc

```diff
@@ -46,8 +48,6 @@ Custom Object Stores must specify an `objectStore` attribute. These Object Store
 
 == See Also
 
-xref:object-store/object-store-to-define-a-new-os.adoc[To Define an Object Store]
-
-xref:object-store/object-store-to-store-and-retrieve.adoc[To Store and Retrieve information from an Object Store]
-
-xref:object-store/object-store-to-watermark.adoc[Example: Watermark]
\ No newline at end of file
+* xref:object-store/object-store-to-define-a-new-os.adoc[To Define an Object Store]
```

Comment:
Should these infinitive headings be tszuj?

Reply:
I'll do that since I'm in anyway!